Flaky test: test_table_data_support[chromium] (nw_table row count 0)

Failed on PR #2349 CI (unrelated docs-only change), passed on rerun.

**Job (attempt 1):** https://github.com/posit-dev/py-shiny/actions/runs/29024300268/job/86139648841 — `playwright-shiny (3.10, chromium, 3)`

**Error:**
```
>       controller.OutputTable(page, "nw_table").expect_nrow(2)
FAILED tests/playwright/shiny/components/table/test_table.py::test_table_data_support[chromium] - AssertionError: Locator expected to have count '2'
  - Expect "to_have_count" with timeout 5000ms
       - unexpected value "0"
```

--- a/tests/playwright/shiny/components/table/test_table.py
+++ b/tests/playwright/shiny/components/table/test_table.py
@@ -9,7 +9,9 @@ from shiny.run import ShinyAppProc
 def test_table_data_support(page: Page, local_app: ShinyAppProc) -> None:
     page.goto(local_app.url)
 
-    controller.OutputTable(page, "nw_table").expect_nrow(2)
+    # The first expectation after page load absorbs the initial render flush,
+    # which can exceed the default 5s timeout on a loaded CI runner.
+    controller.OutputTable(page, "nw_table").expect_nrow(2, timeout=30 * 1000)
     controller.OutputCode(page, "nw_df_type").expect_value(re.compile("narwhals"))
 
     controller.OutputTable(page, "pd_table").expect_nrow(2)

